toolkit-distribution/src/release/LICENSE

```diff
@@ -454,6 +456,7 @@ For details, see their respective project links.
 * swagger-parser (io.swagger:swagger-parser:1.0.44 - http://nexus.sonatype.org/oss-repository-hosting.html/swagger-parser-project/modules/swagger-parser)
 * swagger-parser (org.openapitools.swagger.parser:swagger-parser-core:2.0.14-OpenAPITools.org-1 - http://nexus.sonatype.org/oss-repository-hosting.html/swagger-parser-project/modules/swagger-parser-core)
 * swagger-parser (org.openapitools.swagger.parser:swagger-parser:2.0.14-OpenAPITools.org-1 - http://nexus.sonatype.org/oss-repository-hosting.html/swagger-parser-project/modules/swagger-parser)
+* swagger-parser (org.openapitools.swagger.parser:swagger-parser-core:2.0.14-OpenAPITools.org-1 - http://nexus.sonatype.org/oss-repository-hosting.html/swagger-parser-project/modules/swagger-parser-core)
```

Comment:
It's already declared in line 455: https://github.com/apache/servicecomb-toolkit/blob/master/toolkit-distribution/src/release/LICENSE#L455

Reply:
Ok, I have removed it in the new commit